x-pack/plugins/apm/public/utils/formatters.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { memoize } from 'lodash';
+
+// tslint:disable-next-line  no-var-requires
+const numeral: (input: number) => Numeral = require('@elastic/numeral');
+
+interface Numeral {
+  format: (pattern: string) => string;
+}
+
+const UNIT_CUT_OFF = 10 * 1000000; // 10 seconds in microseconds
+
+export function asSeconds(value: number, withUnit: boolean = true) {
```

Comment:
The type will be inferred when you give it a default value:

```suggestion
export function asSeconds(value: number, withUnit = true) {
```